messages/observations/ais.json

```diff
@@ -0,0 +1,141 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "AIS Vessel",
+    "description": "Data of a specific vessel communicated through AIS messages.",
+    "required": [
+        "sensor_id",
+        "timestamp",
+        "mmsi",
+        "class",
+        "position"
+    ],
+    "properties": {
+        "sensor_id": {
+            "type": "string",
+            "title": "Physical sensor id",
+            "desciption": "Id of the reporting physical sensor equipment"
+        },
+        "timestamp": {
+            "type": "string",
+            "format": "date-time",
+            "title": "Timestamp of AIS message",
+            "description": "Date and time when the message was received, expressed according to the ISO 8601 standard."
+        },
+        "mmsi": {
+            "type": "integer",
+            "title": "MMSI",
+            "description": "MMSI number of the AIS transceiver."
+        },
+        "class": {
+            "type": "string",
+            "enum": [
+                "A",
+                "B"
+            ],
+            "title": "class",
+            "description": "Class of the AIS transceiver, defined by the AIS messages communicating the data of the vessel (A: messages 1,2,3, and 5; B: messages 18, 19, and 24)."
+        },
+        "position": {
+            "$ref": "../core/position.json"
+        },
+        "heading": {
+            "type": "number",
+            "title": "Heading",
+            "description": "Heading of the vessel with respect to True North [degrees]"
+        },
+        "sog": {
+            "type": "number",
+            "title": "SOG",
+            "description": "Speed Over Ground of the vessel [knots]."
+        },
+        "cog": {
+            "type": "number",
+            "title": "COG",
+            "description": "Course Over Ground of the vessel with respect to True North [degrees]"
+        },
+        "rate_of_turn": {
+            "type": "number",
+            "title": "Rate of turn",
+            "description": "Rate of turn of the vessel [degrees/minute]"
+        },
+        "imo_number": {
+            "type": "integer",
+            "title": "IMO number",
+            "description": "IMO number of the vessel"
+        },
+        "draught": {
+            "type": "number",
+            "title": "Draught",
+            "description": "Draught of the vessel"
+        },
+        "navigational_status": {
+            "type": "string",
+            "enum": [
+                "Under way using engine",
+                "At anchor",
+                "Not under command",
+                "Restricted maneuverability",
+                "Constrained by her draught",
+                "Moored",
+                "Aground",
+                "Engaged in fishing",
+                "Under way sailing",
+                "Reserved",
+                "Reserved",
```

Comment:
Dubble trubble

Reply:
Hmm, perhaps I should use the second options for those fields instead:
High-speed craft
Wing-in-ground

Do you know if those are universally used?

Reply:
Changed to use those.